frontends/mit-open/src/common/urls.ts

```diff
@@ -41,3 +41,5 @@ export const login = ({
   const next = `${pathname}${encodeURIComponent(search)}`
   return `${LOGIN}?next=${next}`
 }
+
+export const DASHBOARD = "/dashboard/"
```

Comment:
I feel path constants should not have a trailing slash so we can concatenate predictably, though notice the backend 301s where it's missing.